src/Files.App/UserControls/TabView/TabView.xaml.cs

```diff
@@ -2,45 +2,75 @@
 // Licensed under the MIT License. See the LICENSE.
 
 using CommunityToolkit.WinUI.UI;
-using Files.App.Data.Commands;
-using Files.App.Extensions;
-using Files.App.Helpers;
-using Files.App.ViewModels;
 using Microsoft.UI.Xaml;
 using Microsoft.UI.Xaml.Controls;
 using Microsoft.UI.Xaml.Shapes;
 using Windows.ApplicationModel.DataTransfer;
 using Windows.Storage;
 
-// The User Control item template is documented at https://go.microsoft.com/fwlink/?LinkId=234236
-
-namespace Files.App.UserControls.MultitaskingControl
+namespace Files.App.UserControls.TabView
 {
-	public sealed partial class HorizontalMultitaskingControl : BaseMultitaskingControl
+	public sealed partial class TabView : BaseTabView
 	{
-		public static event EventHandler<TabItem?>? SelectedTabItemChanged;
+		private readonly ICommandManager Commands = Ioc.Default.GetRequiredService<ICommandManager>();
```

Comment:
```suggestion
		private readonly ICommandManager _commands = Ioc.Default.GetRequiredService<ICommandManager>();
```

Reply:
That field is used in xaml so Pascal case would be prefered.